dropwizard-jetty/src/test/java/io/dropwizard/jetty/setup/ServletEnvironmentTest.java

```diff
@@ -90,8 +90,7 @@ public void addsFilterInstances() throws Exception {
         assertThat(holder.getValue().getName())
                 .isEqualTo("filter");
 
-        assertThat(holder.getValue().getFilter())
-                .isEqualTo(filter);
+        assertThat(holder.getValue()).hasFieldOrPropertyWithValue("_instance", filter);
```

Comment:
Accounts for https://github.com/eclipse/jetty.project/commit/d48c258deb54af200636311240a513cf1665354c

Reply:
Seems awfully fragile to rely on internal fields like that.

Reply:
That approach was fine when they updated the master branch to newer Jetty, cf. https://github.com/dropwizard/dropwizard/blob/master/dropwizard-jetty/src/test/java/io/dropwizard/jetty/setup/ServletEnvironmentTest.java#L91

And given my limited time, that will have to do here as well.